dbt_subprojects/hourly_spellbook/models/_sector/perpetual/projects/gains_network/gains_network_base_perpetual_trades.sql

```diff
@@ -0,0 +1,46 @@
+{{ config(
+    schema = 'gains_network_perpetual_trades',
+    alias = 'perpetual_trades',
```

Comment:
Please fix the schema:
```suggestion
    schema = 'gains_network',
    alias = 'perpetual_trades',
```